### PR TITLE
CRM adding: select list of movies

### PR DESCRIPTION
**Pull request by jeremie74**

### CODE REVIEW

File: libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+:host {
+    display: flex;
+    flex-direction: column;
+    max-width: 1135px;
```
Comment (pldespaigne) [inline libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.scss:4]: please use a multiple of 4 for the px

File: libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.ts [hunk 2]
```diff
@@ -7,6 +11,12 @@ import { Component, ChangeDetectionStrategy } from '@angular/core';
...
 })
 export class ContractFormComponent {
+  form = new FormGroup({
+    titleId: new FormControl(),
+  })
+  titles$ = this.service.valueChanges(ref => ref.where('app.catalog.status', '==', 'approved'));
 
+  constructor(private service: MovieService){}
 
+  save() {}
```
Comment (pldespaigne) [inline libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.ts:21]: It's weird to have an empty method like that.
If I wasn't aware that you are still working on this page, I would ask you to delete it

**Review by pldespaigne — APPROVED** [review]

File: libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.ts (hunk 2 above)
Comment (pldespaigne) [reply libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.ts:21]: I just look at the CI logs and this empty method is caught as an error, so will need to remove it to be able to merge

File: libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.scss (hunk 1 above)
Comment (jeremie74) [reply libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.scss]: Done

File: libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.ts (hunk 2 above)
Comment (jeremie74) [reply libs/admin/src/lib/admin-panel/pages/offers/shell/contract-form/contract-form.component.ts]: Done. It was a request from François, i think for the future to be ready.

**Pull request merged**